### ISSUE TITLE
High learning rate

### ISSUE THREAD

**Issue by xsacha** [MccreeZhao/QAMFace#3]
I haven't seen your paper yet but was just wondering how it was determined to use an initial LR of 0.2 instead of the typical 0.1.
While training IR_152 backend, I saw training loss jump significantly to 100.
Also, have you tried retraining using a pretrained backbone?

**Comment by MccreeZhao** [comment]
Hi，thanks for your questions.
In my experiment, I just found that 0.2 performs a little better than 0.1. The difference is not that large.
Since I only have four GPU cards, it's hard for me to train a deeper backbone. All my experiments are based on IR_50. Therefore, I have no idea about the hyperparameters for IR_152. Now I am trying to train a deeper model and I will share my experience with you.
Finally, I haven't tried to retrain a pre-trained model.

**Comment by MccreeZhao** [comment]
![image](https://user-images.githubusercontent.com/26180075/73116204-4fbd1b80-3f6d-11ea-8d81-9a62f49a6043.png)
I tried IR_152 and the training loss keep decrease in the first 1K iterations. The batch size is reduced to 196 due to the lack of GPU memory. What about your hyperparameters?

**Comment by xsacha** [comment]
I didn't get as far to see it decrease like this. I saw the first spike (100-250k iterations) and thought something went wrong.
Thanks. I just need to set first stage to 4th epoch and then it all works from there.
